src/ui/shared/Reply.tsx

```diff
@@ -10,14 +10,29 @@ import {
 import * as React from 'react';
 
 import ErrorToast from 'fontoxml-feedback/src/ErrorToast';
-import { BusyState } from 'fontoxml-feedback/src/types';
+import {
+	BusyState,
+	CardContentComponentProps,
+	Reply as ReplyType
```

Comment:
Why do you rename the reply type? I would directly leave it as Reply.

Reply:
Having a type whose name is the same as the name of a function causes errors:
![image](https://user-images.githubusercontent.com/36703917/167372481-12441656-c0ab-401e-a12b-22fbd6728235.png)